content/en/docs/instrumentation/js/serverless.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Serverless
+weight: 8
+---
+
+This guide will show you how to get started with tracing serverless functions using OpenTelemetry instrumentation libraries.
+
+## AWS Lambda
+
+Following the next steps will allow your AWS Lambda functions to use OpenTelemetry to send traces to the configured backend.
+
+### Dependencies
+
+Create an empty package.json:
+
+```sh
+npm init -y
+```
+
+Install dependencies used by the example.
+
+```sh
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/auto-instrumentations-node \
+  @opentelemetry/exporter-trace-otlp-http \
+  @opentelemetry/instrumentation \
+  @opentelemetry/sdk-trace-base \
+  @opentelemetry/sdk-trace-node
+```
+
+### AWS Lambda wrapper code
+
+This file will contain all the OpenTelemetry logic, which will enable tracing.
+Please save the following code as `lambda-wrapper.js`.
```

Comment:
This involves reproducing a substantial portion of the OTel Lambda layers.  Should the documentation provide guidance on how to use those layers instead of directing the user to create their own wrapper?

Reply:
Hmmm. Would there be any benefit to _not_ configuring and using the layer? I would say yes, we should show that, but I would want a good accounting of the pros and cons.

Reply:
I tried making the documentation as simple as possible to get the user started with tracing their Lambdas, letting them know the hows and the whys.
Using the layer is also possible, but it has to be compiled and deployed by the user.

Reply:
Via the [AWS Distro docs](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) the description _"Users can enable and disable OpenTelemetry for their Lambda function without changing code."_ sounds like we have a "manual vs automatic instrumentation" situation here? 

Agreed with @cartermp we need to have a good understanding of the Pros/Cons of using the one vs the other, that makes it easier to decide if we should have both or only one of them here in the docs. 

If the answer is "both" for the sake of closing this PR we could start with an extra section that calls out the existence of the layer, would that work?

Reply:
I think this documentation should at the least call out the existence of the layers and note that it duplicates a subset of their functionality.  It's good to have the DIY option spelled out, but I expect that many users are not going to want to maintain this themselves if an alternative is available.

Reply:
> I think this documentation should at the least call out the existence of the layers and note that it duplicates a subset of their functionality. It's good to have the DIY option spelled out, but I expect that many users are not going to want to maintain this themselves if an alternative is available.

Makes sense. @samimusallam can you put a pointer to the layers for now. In a follow up we can maybe have both in the docs.

Reply:
I've added a reference to it at the beginning of the docs.